app/views/case_contacts/index.html.erb

```diff
@@ -13,3 +13,11 @@
     </div>
   </div>
 <% end %>
+<% unless @presenter.case_contacts.any? %>
+  <div class="card card-container">
+    <div class="card-body">
+      You have no case contacts for this case. Please click New Case Contact button above to create a case contact for
+      your youth!
```

Comment:
I've seen some people put this text in ` config/locales/views.en.yml ` for localization support. Could make for a good beginner issue.

Reply:
I'm gonna go ahead and make an issue of this